docs/docs/migration-guide.mdx

```diff
@@ -95,12 +120,12 @@ policies:
 
 The new configuration would then look like:
 
-```yaml-rasa
+```yaml-rasa title="config.yml"
 policies:
-# Other policies
-- name: RulePolicy
-  core_fallback_threshold: 0.3
-  core_fallback_action_name: "action_default_fallback"
+  # Other policies
```

Comment:
do we have an convention how to indent yaml in the docs? We usually don't add the 2 spaces afaik

Reply:
I took a look at the config files in the repository and most of them have the 2-space indent. We do not have linting enforcing this at the moment, but we could, I think.